man/git-reauthor.md

```diff
@@ -66,6 +66,10 @@ Change only the committer email of Jack (keep the author email already used)
 
     $ git reauthor --old-email jack@perso.me --correct-email jack@work.com --type committer
 
+Change only the name and keep email to merge one user with multiple pseudonyms.
+
+    $ git reauthor --old-email jack@perso.me --correct-email jack@perso.me --correct-name Jack
```

Comment:
The `--correct-email jack@perso.me` part is useless. If you want to keep the email part, use `--correct-name blahblah` is enough.

Reply:
Good catch. Makes it simpler too. Change made.